pkg/runner/chroot_runner.go

```diff
@@ -0,0 +1,73 @@
+package runner
```

Comment:
Is there a way we could get unit testing coverage for this type?

Using “pkg/filesystem” instead of “os” might be helpful to achieve that.